core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/views/LegacyYouTubePlayerView.kt

```diff
@@ -98,6 +100,11 @@ internal class LegacyYouTubePlayerView(context: Context, attrs: AttributeSet? =
         if (handleNetworkEvents)
             context.registerReceiver(networkListener, IntentFilter(ConnectivityManager.CONNECTIVITY_ACTION))
 
+        if(shouldUseWebUi){
+            inflateCustomPlayerUi(R.layout.ayp_empty_layout)
+            isUsingWebUi = true
+        }
+
```

Comment:
Why? Was I doing this before to enable web ui?

Reply:
yeah